### DISCUSSION TITLE
jupyter notebook not found pada tugas capstone

### DISCUSSION THREAD

**Discussion by dinaf39** [teamalgoritma/community#657] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/130343206/0505b215-c4c8-4b9d-923d-9aa02769349b)

kenapa ya jupyter notebook not found, dan jika saya buka langsung dari app jupyter note book tanpa buka anaconda promt, read csv nya tidak terbaca

**Comment by kevbow** [comment, marked as answer]
Dear @dinaf39,

Hal ini karena Anda tidak melakukan instalasi jupyter pada virtual environment `capstone-flask`, silakan membuka jupyter notebook melalui base environment.
Sementara untuk read_csv yang tidak terbaca, pastikan Anda sudah melakukan importing library pandas sebelumnya.
Semoga membantu.

**Reply by dinaf39** [reply]
![image](https://github.com/teamalgoritma/community/assets/130343206/4c219545-3fff-4aed-902e-9f6b99413173)
sudah saya lakukan import pandas as pd, tapi tidak bisa juga, apakah ada step yang salah

**Reply by kevbow** [reply]
Bisa coba ditunjukkan pesan error paling bawahnya Bu?

**Reply by dinaf39** [reply]
![image](https://github.com/teamalgoritma/community/assets/130343206/ea28a0eb-76bd-46ac-8389-8ac80de11aa7)
![image](https://github.com/teamalgoritma/community/assets/130343206/0093c80e-6ff9-454b-bb5d-50fc01cd5ebf)
![Uploading image.png…]()

**Comment by kevbow** [comment]
Pesan yang keluar adalah tidak terdapat file yang ingin Anca baca. Pastikan kembali file .ipynb ini ada pada folder yang sama dengan folder data_input.

**Reply by dinaf39** [reply]
![image](https://github.com/teamalgoritma/community/assets/130343206/1ee57336-3183-4cf5-b569-c9f0a8414151)
sudah saya letakan data ditempat yang sama namun tetap tidak terbaca

**Reply by dinaf39** [reply]
![image](https://github.com/teamalgoritma/community/assets/130343206/a96c47a3-7dfc-494d-aa95-3ef3e2b31f47)
![image](https://github.com/teamalgoritma/community/assets/130343206/aba50a34-4667-446f-a608-ddd86251d728)
![image](https://github.com/teamalgoritma/community/assets/130343206/b98e39a5-876c-499e-9362-b371916bf149)
![image](https://github.com/teamalgoritma/community/assets/130343206/d7f5b9f1-2e3e-4887-950a-cd5cff6455ab)

**Reply by kevbow** [reply]
Hal ini karena Anda menyimpan file pada folder `data` sementara yang Anda tuliskan pada read_csv adalah `data_input`.

**Reply by dinaf39** [reply]
terima kasih banyak mas bantuannya, sdh bisa
